logging-api/src/main/java/com/cisco/oss/foundation/logging/transactions/SchedulerLogger.java

```diff
@@ -47,6 +56,20 @@ public static void success(String schedulerResult) {
 
     }
 
+    public static void successAsync(SchedulerLogger schedulerLogger) {
+
+        FlowContextFactory.deserializeNativeFlowContext(TransactionLogger.getFlowContextAsync(schedulerLogger));
```

Comment:
a TransactionLogger instance holds the FCID as a string -- serialized. successInstance() invoked here prints the FCID using log4j, from the MDC. what the logic here does, as I understand it, is to put the FCID in the MDC right before printing. 
I agree the design is confusing. Do you have a suggestion as to how to change it?